src/oscar/apps/catalogue/reviews/abstract_models.py

```diff
@@ -6,11 +6,13 @@
 from django.utils.translation import ugettext_lazy as _
 from django.utils.translation import pgettext_lazy
 
-from oscar.apps.catalogue.reviews.managers import ProductReviewQuerySet
-from oscar.apps.catalogue.reviews.utils import get_default_review_status
 from oscar.core import validators
+from oscar.core.loading import get_class
 from oscar.core.compat import AUTH_USER_MODEL, user_is_authenticated
 
+ProductReviewQuerySet = get_class('catalogue.reviews.managers', 'ProductReviewQuerySet')
+get_default_review_status = get_class('catalogue.reviews.utils', 'get_default_review_status')
```

Comment:
This isn't actually a class - I don't think we should make loading of methods dynamic. If someone needs to override this default they should override the model field that uses it I think?